docs/_docs/windows.md

```diff
@@ -177,7 +177,27 @@ In the future the installation process of the github-pages should be as simple a
 RubyInstaller is a self-contained Windows-based installer that includes the Ruby language, an execution environment, important documentation, and more.
 
 1. Install a package manager for Windows called [RubyInstaller](https://rubyinstaller.org/).
-2. Install Jekyll via RubyInstaller: `gem install bundler jekyll`
-3. Reopen a command prompt and run Jekyll: `jekyll -v`
+2. Install Jekyll and Bundler via a command prompt window: `gem install jekyll bundler`
+3. Check if the installation is accessible: `jekyll -v`
 
 see [Autoinstall Jekyll for windows](https://github.com/KeJunMao/fastjekyll#autoinstall-jekyll-for-windows)
+
+### auto-regeneration
+
+Although jekyll would suggest:
+
+```
+Please add the following to your Gemfile to avoid polling for changes:
+    gem 'wdm', '>= 0.1.0' if Gem.win_platform?
+```
+
+But auto-regeneration still works fine.Its an outdated warning IMO.
+
+### time-zone
+
+Requires one gem:
+
+```shell
+gem install tzinfo-data
```

Comment:
@ashmaroli 

Change to: ?

### auto-regeneration

Although jekyll would suggest:

```
Please add the following to your Gemfile to avoid polling for changes:
    gem 'wdm', '>= 0.1.0' if Gem.win_platform?
```

Auto-regeneration will work fine without including `gem 'wdm'`

### time-zone

Though `tzinfo-data` is required, you don't have to install it separately.
It is already included in a new Gemfile generated by `jekyll new` from `v3.4.0` For sites generated with older versions, they **should** add it to their Gemfile,Then run the `bundle install`.

```
# Windows does not include zoneinfo files, so bundle the tzinfo-data gem
gem 'tzinfo-data', platforms: [:mingw, :mswin, :x64_mingw, :jruby]
```

More about timezone-management see [here](/docs/windows/#timezone-management).